frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -2597,6 +2626,7 @@
             vm.stopFetchingSubmissions();
             vm.stopLeaderboard();
             vm.stopLoadingLogs();
+            vm.stopFetchingSubmissions();
```

Comment:
@burnerlee this looks like an incorrect method call? Why do we have to stop fetching submissions?